src/lang/english.txt

```diff
@@ -2380,6 +2380,7 @@ STR_NETWORK_SERVER_LIST_SERVER_ADDRESS                          :{SILVER}Server
 STR_NETWORK_SERVER_LIST_INVITE_CODE                             :{SILVER}Invite code: {WHITE}{RAW_STRING}
 STR_NETWORK_SERVER_LIST_START_DATE                              :{SILVER}Start date: {WHITE}{DATE_SHORT}
 STR_NETWORK_SERVER_LIST_CURRENT_DATE                            :{SILVER}Current date: {WHITE}{DATE_SHORT}
+STR_NETWORK_SERVER_LIST_PLAY_TIME                               :{SILVER}Play time: {WHITE}{NUM} hours
```

Comment:
Would it make sense to use `{DECIMAL}`, so you can differentiate between "running a few minutes" vs "running for almost an hour"? In some competitive games that might be quite important.

I'm myself still really wondering whether it would make sense, so maybe some others can give their opinions about it before making any changes.

Reply:
Ideally, I would have it say `1d12h3m` .. but that went a bit far for this PR :D

Decimals are a bit annoying, as 1.25 is 1h15m .. and I find that confusing to read. So I rather avoid that.

Reply:
Going with: `Play time: 12 hours, 43 minutes`

That should work (for now) :)